src/components/map/hooks/use-tree-circle-style.tsx

```diff
@@ -2,12 +2,21 @@
 import { Expression } from "mapbox-gl";
 import { useMapConstants } from "./use-map-constants.js";
 import resolveConfig from "tailwindcss/resolveConfig";
+import { isWithinInterval } from "date-fns";
 
 //@ts-expect-error tailwindConfig has no type definition
 import tailwindConfig from "../../../../tailwind.config.js";
 import { TreeAgeRange } from "../../filter/filter-store";
 const fullConfig = resolveConfig(tailwindConfig);
 
+const currentDate = new Date();
+const currentYear = currentDate.getFullYear();
+
+const isInVegetationPeriod = isWithinInterval(currentDate, {
+	start: new Date(currentYear, 2, 1), // Beginning of March
+	end: new Date(currentYear, 8, 31), // End of October
```

Comment:
the vegetation period is set to end of september atm for testing. I will set it to end: new Date(currentYear, 9, 31) before merging

Reply:
Just a minor detail, that might be irrelevant: if you use declare the end date like this, it will use the begin of the day (00:00:00). Meaning `isInVegetationPeriod` will be falsy already on the 31. October. If you need the end of the day you also need to define the hours, minutes etc, or use something like this https://date-fns.org/v4.1.0/docs/endOfMonth 👍

Reply:
Ah good to know! 👍